include/pokeagb/graphics/background.h

```diff
@@ -30,6 +30,14 @@ struct BgConfig2 {
     u32 y;
 };
 
+struct mapblock_16color {
```

Comment:
Rename this to fit the naming convention (Pascal case for structs) and to reflect the fact that it's a tilemap struct, e.g. `TileMapBlock4BPP`